tests/test_bag_brain_io.py

```diff
@@ -7,14 +7,14 @@
 
 
 class TestBagBrainIO(unittest.TestCase):
-    test_bag = "/home/jake/bagfiles/test_bags/record-brain-freshplant18200-samp0.0_2021-08-27-15-49-21_1.bag"
-    test_onnx = "/home/jake/bagfiles/test_bags/freshplant18200.onnx"
+    test_bag = "/home/jake/bagfiles/test_bags/record-brain-sac-pleasant-frog-450-02048-samp0.0_2021-09-06-13-24-24_3.bag"
+    test_onnx = "/home/jake/hipow-checkpoints/sac-pleasant-frog-450-02048.onnx"
     test_yolo_onnx = os.path.join(os.path.dirname(__file__), "..", "yolov5s.onnx")
 
     def test_lstm_sample(self):
         bag = rosbag.Bag(self.test_bag, "r")
         onnx = rt.InferenceSession(self.test_onnx)
-        history_size = 8
+        history_size = 240
```

Comment:
=O

Reply:
Can you explain this change?

Reply:
When you export ONNX, you need to pass sample tracing data into it that includes at least as many elements as you reference.